test/test-deployment-redirects.js

```diff
@@ -18,30 +18,45 @@ const chalk = require('chalk')
 
 const argv = require('minimist')(process.argv.slice(2))
 
-const input = argv.input || './encore-vega-redirects.csv'
+const input = argv.input || './redirect-expectations.csv'
+
+const changeToQa = (url) => {
+  return url
+    // Prod Encore to Test Encore:
+    // Note this has another valid hostname: https://nypl-encore-test.
```

Comment:
Are we maintaining coverage for `borrow.nypl.org` still?

Reply:
This app doesn't handle traffic on `borrow.nypl.org`, but does issue redirects to that domain. Do you mean `browse.nypl.org`?

Reply:
oh yes i mixed up the browse and borrow. borrow is the new domain but that's being handled w reverse proxy biz?

Reply:
Yes `borrow` is the new domain. I believe it's just a CNAME pointing `borrow.nypl` to the Innovative servers.